torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13002,8 +12902,6 @@ def ref_pairwise_distance(input1, input2):
             DecorateInfo(unittest.skip("Skipped!"), 'TestReductions', 'test_dim_none'),
             DecorateInfo(unittest.skip("Skipped!"), 'TestReductions', 'test_dim_none_keepdim'),
             # FIXME: improve precision
-            DecorateInfo(unittest.skip("Skipped!"), 'TestReductions', 'test_noncontiguous_all',
```

Comment:
Interesting... what fixed this?

Reply:
I believe it was failing due to some problems in the previous generation of the inputs (perhaps some flags of the final tensor were not set properly). After rewriting the sampling I run all these to see if it had fixed any of the problems and...